gym-socialgame/gym_socialgame/envs/buffers.py

```diff
@@ -79,64 +79,65 @@ def entropy(self):
 class GaussianBuffer(BaseBuffer):
     def __init__(self, obs_dim):
         super().__init__()
-        self.buffer = np.zeros((1,obs_dim))
+        self.buffer = np.zeros((100,obs_dim))
         self.buffer_size = 1
         self.obs_dim = obs_dim
-        self.add(np.ones((1,obs_dim)))
-        self.add(-np.ones((1,obs_dim)))
-        
+        self.add(np.ones(obs_dim))
+        self.add(-np.ones(obs_dim))
+
     def add(self, obs):
-        self.buffer = np.concatenate((self.buffer,obs.flatten()[np.newaxis,:]), axis=0)
+        self.buffer[self.buffer_size] = obs.flatten()
         self.buffer_size += 1
+        if self.buffer_size >= self.buffer.shape[0]:
+            self.buffer = np.append(self.buffer, np.zeros(self.buffer.shape), axis=0)
 
     def get_params(self):
         #import pdb; pdb.set_trace()
-        means = np.mean(self.buffer, axis=0)
-        stds = np.std(self.buffer, axis=0)
+        means = np.mean(self.buffer[:self.buffer_size], axis=0)
+        stds = np.std(self.buffer[:self.buffer_size], axis=0)
```

Comment:
Why do we make a copy of the buffer? Especially since we're just calculating means and stds?